README.md

```diff
@@ -188,6 +190,61 @@ $ curl -X DELETE --url vmpooler.company.com/vm/fq6qlpjlsskycq6
 }
 ```
 
+#### Status and metrics
+
+##### GET /status
+
+A "live" status endpoint.
```

Comment:
Can you mention that this is the current view or current state of the pooler? The `/summary` portion will then be sure to mention the historical nature / purpose.